app/src/main/java/com/projectit3048c/MainViewModel.kt

```diff
@@ -10,7 +10,7 @@ import kotlinx.coroutines.launch
 import com.projectit3048c.service.FoodService
 import com.projectit3048c.service.IFoodService
 
-class MainViewModel(var foodService : IFoodService =  FoodService()) : ViewModel() {
+class MainViewModel(var foodService: IFoodService =  FoodService()): ViewModel() {
```

Comment:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.